### ISSUE TITLE
Fix the funnel crouch flying exploit.

### ISSUE THREAD

**Issue by FrozenDragon0** [ChaosInitiative/Portal-2-Community-Edition#259]
Title says it all. This damn bug needs to be patched once and for all.

**Comment by L89David** [comment]
No, this cannot be patched. You don't think about speedrunners?

**Comment by SpyceTewan** [comment]
stop spamming our issues about topics we have discussed a million times

**Comment by Luke18033** [comment]
But any speedrun would be required to use the base game, right? Which makes this kind of pointless. *Maybe* leave it in as a cvar, but even then it should be off by default.

**Comment by FrozenDragon0** [comment]
Actually now that @Luke18033 mentions it having this bug as a cvar wouldn't be a bad idea.

**Comment by Konclan** [comment]
Would still like to see a cvar that disables funnel flying by default, which could be enabled by speedrunners like #724. The major issue I have with funnels is that sometimes even when not trying to do the funnel fly glitch players will sometimes inexplicably float outside of a funnel way longer than normal causing them to go distances that shouldn't be possible. This bug is more subtle than normal funnel flying and some players don't notice it happening causing them to reach areas that are normally barely unreachable. For this reason I would like to see funnel checks improved.

**Comment by Luke18033** [comment]
Indeed, at the time it hadn't really been decided how speedrunning bugs would be handled. This will be controlled by a cvar.

**Comment by Luke18033** [comment]
Why?

**Comment by EchoesForeAndAft** [comment]
It's been discussed already this will be patched but available to speedrunners via a special speedrunning mode/cvar.

**Comment by Luke18033** [comment]
That's what this issue was for.
